spec/models/manageiq/providers/ansible_tower/automation_manager/refresher_spec.rb

```diff
@@ -116,18 +116,18 @@ def assert_configuration_script_with_survey_spec
     expect(system).to have_attributes(
       :name        => "Ansible-JobTemplate-Survey",
       :description => "Ansible-JobTemplate-Description",
-      :manager_ref => "155",
+      :manager_ref => "81",
       :variables   => {'abc' => 123}
     )
     survey = system.survey_spec
     expect(survey).to be_a Hash
-    expect(survey['spec'].first['index']).to eq 0
```

Comment:
We don't create them and I don't see any need to create them any time soon.  They are used by the Tower UI to get launch data from users.  We parse any existing surveys to create dialogs in ManageIQ, but I don't see index being important there either.